scripts/nmt/nmt.py

```diff
@@ -55,7 +55,6 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--tgt-language", type=str, help="Target language (according to BCP-47 standard)"
     )
-    parser.add_argument("--text-file", type=str, help="Path to file for translation")
```

Comment:
Why do we remove this? If we do, then we need to remove code using `args.text_file`

Reply:
This commit got picked while merging the branch. Let me remove it and keep only the "Require setuptools>=65" commit.